en/lessons/gravity-model.md

```diff
@@ -0,0 +1,712 @@
+
+**Numerical Data Only**
+
+As the gravity model is a mathematical equation, all input variables must be numerical. That could be a count (population), spatial measure (area, distance, etc), time (hours from London on foot), percentage (wage increase/decrease), currency value (wages in shillings), or some other *measure* of the places involved in the model.
+
+Numbers must be meaningful and cannot be nominal [categorical variables](https://en.wikipedia.org/wiki/Categorical_variable) which act as a stand-in for a qualitative attribute. For example you **cannot** arbitrarily assign a number and use it in the model if the number doesn't have meaning (eg, road quality = good, or road quality = 4). Though the latter is numerical it is not a measure of road quality. Instead, you might use the average travel speed in miles per hour as a proxy for road quality. Whether average speed is a meaningful measure of road quality is up to you to determine and defend as the author of the study.
+
+Generally speaking, if you can measure it or count it, you can model it.
+
+**Complete Data Only**
+
+All categories of data must exist for each point of interest. That means that all of the 32 counties under analysis must have reliable data for each push and pull factor. You cannot have any gaps or blanks, such as one county where you don't have the average wage.
+
+**Reliable Data Only**
+
+The computer science adage "garbage in, garbage out" also applies to gravity models, which are only as reliable as the data used to build them. Beyond choosing robust and reliable historical data from sources you can trust, there are lots of ways to make mistakes that will render the outputs of your model meaningless. For example, it is worth making sure that the data you have exactly match the territories (eg, county data to represent counties, not city data to represent a county). 
+
+Depending on the time and place of your study, you may find it difficult to obtain a reliable set of data upon which to base your model. The further back in the past one's study, the more difficult that may be. Likewise, it may be easier to conduct these types of analyses in societies that were heavily bureaucratic and left a good surviving paper trail, such as in Europe or North America.
+
+To ensure data quality in this case study, each variable was either reliably calculated or derived from published peer-reviewed historical data (see Table 1). Exactly how these data were compiled can be read in the original article where it was explained in depth.[^11]
+
+### Our Five Model Variables
+
+With the above principles in mind, we could have chosen any number of variables, given what we knew about migration push and pull factors. We settled on FIVE, chosen based on what we thought would be most important, and which we knew could be backed up with reliable data.
+
+<tablecaption>Table 1: The Five variables used in the model, and the source of each in the peer reviewed literature.</tablecaption>
```

Comment:
`<tablecaption>` isn't a real html tag! Unfortunately the markdown converter doesn't incorporate figure captions for us :( The real solution for adding table captions will be to [convert the markdown tables into regular HTML](https://www.tablesgenerator.com/html_tables) and then add a `<caption>` tag just inside the `<table>` tag as described here: <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption>

Reply:
I don't think it's a good idea for me to use HTML tables here, since our instructions specifically say to use Markdown tables. Is there another solution?

Reply:
I've changed these to <caption>. I realize that isn't right by HTML standards because it should be immediately after <table>, but given that we use markdown, I'm not sure what else I can do.

Reply:
No, sorry for the confusion. What I meant was that you should convert the tables into HTML and put `<caption>` immediately inside the `<table>` tag - you can drop HTML into markdown and it will render just fine. We do it all the time in order to get things like the `<div class="alert">` things showing up in lessons like https://programminghistorian.org/en/lessons/introduction-to-ffmpeg#installing-ffmpeg

Reply:
You can paste the current markdown tables into https://www.tablesgenerator.com/html_tables under "File > Paste Data..." and it will render it into HTML for you. Make sure to check "Do not generate CSS"

Reply:
done